dotcom-rendering/scripts/gen-stories/get-stories.mjs

```diff
@@ -2,7 +2,9 @@
 
 /** @fileoverview
  *
- * Use: node dotcom-rendering/scripts/gen-stories/gen-stories.js
+ * Use: node dotcom-rendering/scripts/gen-stories/gen-stories.mjs
+ *
+ * Run: make gen-stories in the dotcom-rendering sub-directory
```

Comment:
I don’t think we should tell people how to run a JS file with node, especially if it risks becoming out of sync as this file moves around.

```suggestion
 * Run: `make gen-stories` in the dotcom-rendering sub-directory
```